model_tools/brain_transformation/behavior.py

```diff
@@ -215,3 +215,47 @@ def labels_to_indices(self, labels):
                 indices.append(label2index[label])
             index2label = OrderedDict((index, label) for label, index in label2index.items())
             return indices, index2label
+        
+
+class OddOneOutBehavior(BrainModel):
+    def __init__(self, identifier, activations_model, layer):
```

Comment:
```suggestion
    def __init__(self, identifier: str, activations_model, layer: Union[str, List[str], similarity_measure: str = 'dot']):
```
let's add some type hints and specify the similarity_measure